src/pages/saksbehandling/avslag/AvslåttSøknad.tsx

```diff
@@ -0,0 +1,94 @@
+import * as RemoteData from '@devexperts/remote-data-ts';
+import { Alert, Button, Link, Loader, Textarea } from '@navikt/ds-react';
+import React from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useHistory } from 'react-router-dom';
+
+import { AvslagManglendeDokType } from '~api/søknadApi';
+import ApiErrorAlert from '~components/apiErrorAlert/ApiErrorAlert';
+import { avslagManglendeDokSøknad } from '~features/saksoversikt/sak.slice';
+import { useI18n } from '~lib/i18n';
+import * as Routes from '~lib/routes';
+import { useAppDispatch, useAppSelector } from '~redux/Store';
+import { Sak } from '~types/Sak';
+
+import nb from './lukkSøknad-nb';
+import styles from './lukkSøknad.module.less';
+
+interface Props {
+    sak: Sak;
+}
+
+const AvslåttSøknad = (props: Props) => {
+    const dispatch = useAppDispatch();
+    const { control, handleSubmit } = useForm<AvslagManglendeDokType>();
+    const { søknadLukketStatus, lukketSøknadBrevutkastStatus } = useAppSelector((s) => s.sak);
+    const urlParams = Routes.useRouteParams<typeof Routes.avsluttSøknadsbehandling>();
+    const søknad = props.sak.søknader.find((s) => s.id === urlParams.soknadId);
+    const { intl } = useI18n({ messages: nb });
```

Comment:
Bruk gjerne
```suggestion
    const { formatMessage } = useI18n({ messages: nb });
```
da den både er mer praktisk og mer typesikker :)